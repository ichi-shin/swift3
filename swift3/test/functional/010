#!/bin/bash

# DELETE Bucket lifecycle

. ./common

S3USER=tester

_s3_put /bucket

echo '<LifecycleConfiguration>
<Rule>
   <ID>Archive and then delete rule</ID>
   <Prefix>prefix_</Prefix>
   <Status>Enabled</Status>
   <Expiration><Days>0</Days></Expiration>
</Rule>
</LifecycleConfiguration>' > $tmp.lifecycle.xml
MD5=$(_md5 $tmp.lifecycle.xml) _s3_put /bucket?lifecycle -T $tmp.lifecycle.xml

_s3_delete /bucket?lifecycle -D - | _filter_curl
