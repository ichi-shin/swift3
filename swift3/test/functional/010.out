QA output created by 010
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester --contentMd5 U2caJjlmWSTwONBlU/U/dA== -- -X PUT -T /TMP.lifecycle.xml http://SWIFT_HOST/bucket?lifecycle... 200
> s3curl --id tester -- -X DELETE -D - http://SWIFT_HOST/bucket?lifecycle... 204
HTTP/1.1 204 No Content
Content-Length: LENGTH
Content-Type: text/html; charset=UTF-8
Date: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

